flake8_trio/visitors/flake8triovisitor.py

```diff
@@ -228,6 +224,12 @@ def error(
             )
         )
 
+    def autofix(self, code: str | None = None):
```

Comment:
Maybe `should_autofix`?  This seems more like an imperative than a logical test.

Reply:
for sure, changed